app/src/main/java/com/infomaniak/swisstransfer/ui/components/transfer/TransferItemList.kt

```diff
@@ -36,12 +37,14 @@ import com.infomaniak.swisstransfer.ui.utils.PreviewLightAndDark
 @Composable
 fun TransferItemList(
     modifier: Modifier = Modifier,
+    direction: TransferDirection,
     transfers: List<TransferUi>,
     getSelectedTransferUuid: () -> String?,
     onClick: (TransferUi) -> Unit,
 ) {
 
     val selectedTransferUuid = getSelectedTransferUuid()
+    val titleRes = if (direction == TransferDirection.SENT) R.string.sentFilesTitle else R.string.receivedFilesTitle
```

Comment:
Maybe a `when` would be more suitable as it's an enum, so we don't risk a potential bad value in the hypothetical case we add smt in the enum